.git-blame-ignore-revs

```diff
@@ -0,0 +1,4 @@
+# This feature requires Git >= 2.24
+# To use it by default in git blame:
+# git config blame.ignoreRevsFile .git-blame-ignore-revs
+09a4679b9eaafb430d1c19a83e5c3143ea34b745
```

Comment:
Sorry I missed this, but I think this needs to be updated now too.

Reply:
Good catch. I pushed the fix. Can't wait for this to go in so I can stop recreating this changeset...